kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -1015,8 +1166,13 @@ define([
                 }
 
                 if (appInfo.app_output_param) {
+                    let outputName = file;
+                    // if we're in a subpath, need to strip it down to just the file name
```

Comment:
Something similar needs to be done for the bulk import setup, including adding the file type suffix. I'll try to do that next.